packages/types/src/discordeno.ts

```diff
@@ -305,6 +324,76 @@ export interface InputTextComponent {
   value?: string
 }
 
+/** https://discord.com/developers/docs/components/reference#section-section-structure */
+export interface SectionComponent extends BaseComponent {
+  type: MessageComponentTypes.Section
+
+  /** One to three text components */
+  components: TextDisplayComponent[]
+  /** A thumbnail or a button component, with a future possibility of adding more compatible components */
+  accessory: ButtonComponent | ThumbnailComponent
+}
+
+/** https://discord.com/developers/docs/components/reference#text-display */
+export interface TextDisplayComponent extends BaseComponent {
+  type: MessageComponentTypes.TextDisplay
+
+  /** Text that will be displayed similar to a message */
+  content: string
+}
+
+/** https://discord.com/developers/docs/components/reference#thumbnail */
+export interface ThumbnailComponent extends BaseComponent {
+  type: MessageComponentTypes.Thumbnail
+
+  /** A url or attachment */
+  media: DiscordUnfurledMediaItem
```

Comment:
Type of `component.media` is [unfurled media item](https://discord.com/developers/docs/components/reference#unfurled-media-item-structure), which has snake case and null, so do we need to transform it?